packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -184,6 +184,13 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
   }
 
   async close() {
+    if (!this.dispatcher?.transferID) {
+      throw new ProviderTransferError('transferID is not defined');
```

Comment:
```suggestion
      throw new ProviderTransferError('Failed to end the transfer. No transfer process found.);
```

The error will end up in the CLI, I would advise having something clearer for the end user, what do you think?